src/app/download/download.component.spec.ts

```diff
@@ -21,13 +23,38 @@ describe('DownloadComponent', () => {
     loader = TestbedHarnessEnvironment.loader(fixture);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    state = TestBed.inject(AdAuthoringWorkflowStateContainer);
   });
 
   it('should create the download component', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should call downloadFile when the download button is clicked', async () => {
+  it('should expect download button to be disabled', async () => {
```

Comment:
Can you also add a test to make sure Download button is disabled if one of the two pre-reqs is true?